layouts/posts/single.html

```diff
@@ -58,10 +58,12 @@ <h1 class="post-title">
       </div>
     </article>
 
+    {{ partial "likecoin.html" . }}
```

Comment:
What does the `likecoin.html` file do and where is it located? It does not currently exist in this repo.